modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating to a cluster with multi-architecture compute machines using the CLI
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* Your {product-title} version is up to date to at least version 4.13.0. 
++
+For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_.
+* You have installed the OpenShift CLI (`oc`) that matches the version for your current cluster.
+* Your `oc` client is updated to at least verion 4.13.0. 
+* Your {product-title} cluster is installed on either the AWS or Azure platform. 
++
+For more information on selecting a supported platform for your cluster installation, see _Selecting a cluster installation type_.
+
+
+.Procedure
+. Verify that the `RetrievedUpdates` condition is `True` in the Cluster Version Operator (CVO) by running the following command:
++
+[source,terminal]
+----
+$ oc get clusterversion/version -o=jsonpath="{.status.conditions[?(.type=='RetrievedUpdates')].status}"
```

Comment:
as far as the command is concerned, if what we're after is not to expose too much cvo internals, we may as well only ask the user to check oc adm upgrade out, verifying there's no "RetrievedUpates=False" in the output, and if there is, to address it following the message prior to progressing any further. we may as well suggest ignoring "Upgradeable=False", which (overrides aside) should not block the procedure. imho, a lack of RetrievedUpates=False in the output should be good enough to assume RetrievedUpates=True